internal/provider/resource_gitlab_project_runner_enablement.go

```diff
@@ -0,0 +1,144 @@
+package provider
+
+import (
+	"context"
+	"log"
+	"strconv"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	gitlab "github.com/xanzy/go-gitlab"
+)
+
+var _ = registerResource("gitlab_project_runner_enablement", func() *schema.Resource {
+	return &schema.Resource{
+		Description: `The ` + "`gitlab_project_runner_enablement`" + ` resource allows to manage the linking between a project and a runner instance.
```

Comment:
Can we get the wording aligned here, too? `linking` -> rephrase to something with "enable" ...

Reply:
Changed it to
> allows to enable a runner in a project.